client/assets/images/icons/light-bulb-dark.svg

```diff
@@ -0,0 +1 @@
+<svg height="18" viewBox="0 0 14 18" width="14" xmlns="http://www.w3.org/2000/svg"><path d="m8.75005 16.0938h-4.68755c-.08594 0-.15625.0703-.15625.1562v.625c0 .3457.2793.625.625.625h3.75c.3457 0 .625-.2793.625-.625v-.625c0-.0859-.0703-.1562-.1562-.1562zm-2.3438-16.0938c-3.53711 0-6.40625 2.86914-6.40625 6.40625 0 2.37105 1.28906 4.44145 3.20313 5.54885v2.2637c0 .3457.27929.625.625.625h5.15622c.3457 0 .625-.2793.625-.625v-2.2637c1.9141-1.1074 3.2031-3.1778 3.2031-5.54885 0-3.53711-2.8691-6.40625-6.4062-6.40625zm2.498 10.7383-.7011.4062v2.293h-3.5938v-2.293l-.70115-.4062c-1.53711-.8887-2.50195-2.5254-2.50195-4.33205 0-2.76172 2.23828-5 5-5 2.7617 0 5 2.23828 5 5 0 1.80665-.9648 3.44335-2.502 4.33205z" fill="#fff" transform="translate(.59375 .25)"/></svg>
```

Comment:
There look to be some extraneous properties here, I'd recommend
- Removing `height` and `width`
- Setting viewBox to the standard `0 0 24 24`
- Removing `fill` and `transform`

Reply:
Realizing you may need the fill for the `dark` aspect, unless it's possible to use the original icon just change the fill for dark mode via css?

Reply:
I kept height and width (both set to 20) in there because I think it helps ensure the icon doesn't get squished on some browsers. I updated viewbox to be `0 0 20 20`. I removed transform, but kept fill in place.

> Realizing you may need the fill for the dark aspect, unless it's possible to use the original icon just change the fill for dark mode via css?
The SVG would need to be loaded directly into the DOM for us to be able to dynamically change the fill. Since we're adding the icon after the DOM has loaded (using ::before), this option is not available.